reorganize HCQ and add prophylaxis (#369)
greenelab/covid19-review: content/20.treatments.md

However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83, although the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC-alone group (mean of 0.062 versus 0.008; p=0.57).
Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response, which could help to prevent multiorgan failure and death.
However, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC.
The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004).

> The study had a few limitations.

Author: @alavendelm I think I also suggested reworking this text, similar to your feedback in #367

However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83, although the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC-alone group (mean of 0.062 versus 0.008; p=0.57).
Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response, which could help to prevent multiorgan failure and death.
However, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC.
The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004).

> Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.

Reviewer: The adverse effects of administration of HCQ to mild and moderate COVID-19 cases seemed manageable.

> Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.

Author: ```suggestion
The adverse effects of administering HCQ to mild and moderate COVID-19 cases were found to be manageable.
```
